src/ensembl/src/content/app/browser/browserActions.ts

```diff
@@ -395,3 +395,36 @@ export const toggleRegionEditorActive = createAction(
 export const toggleRegionFieldActive = createAction(
   'browser/toggle-region-field-active'
 )<boolean>();
+
+export const clearActiveGenomeId = createAction(
+  'browser/clear-active-genome-id'
+)();
+
+export const clearActiveEnsObjectIdsForGenome = createAction(
+  'browser/clear-active-ens-object-ids-for-genome'
+)<string>();
+
+export const deleteSpeciesAndSaveBrowser = (
```

Comment:
Maybe without `Browser` in the end? You aren't calling `clearActiveGenomeId` action above `clearActiveGenomeIdBrowser` :-)

I understand that we'll have similarly named actions for other apps (EntityViewer, CustomDownload, etc.). I don't think it's going to be a problem.

Reply:
That's done